### PR TITLE
Feat/top bar

### PR DESCRIPTION
**Pull request by Kreezag**
### Fix for #646 
- Add top-bar to app-layout
- Fix top-bar behaviour
- Refactor app layout (fix some bugs with fixed navbar)
- Replace top-bar components from vuestic-ui to vuestic-admin
- Replace navbar components from vuestic-ui to vuestic-admin
- Update stylelint in some files

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/app/main.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ import VueClipboard from 'vue-clipboard2'
 import '../metrics'
```
Comment (asvae) [inline src/app/main.js]: 3 dots off position.
image

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 2]
```diff
@@ -3,21 +3,26 @@
...
   >
-    <div class="app-navbar__icon-container">
-      <span
-        class="app-navbar__menu"
-        :class="`i-menu-${minimized ? 'collapsed' : 'expanded'}`"
-        @click="$emit('update:minimized', !minimized)"
-      ></span>
-    </div>
-
     <div class="app-navbar__content row">
```
Comment (asvae) [inline src/components/admin/app-navbar/AppNavbar.vue]: row twice?

File: src/components/admin/app-navbar/components/dropdowns/ColorDropdown.vue.orig [hunk 3]
```diff
@@ -0,0 +1,142 @@
+<template>
```
Comment (asvae) [inline src/components/admin/app-navbar/components/dropdowns/ColorDropdown.vue.orig]: This file is weird.

File: src/app/main.js (hunk 1 above)
Comment (asvae) [inline src/app/main.js]: Email an github link should be centered.
image

File: src/components/dashboard/Dashboard.vue.orig [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<template>
```
Comment (asvae) [inline src/components/dashboard/Dashboard.vue.orig]: orig

File: src/components/dashboard/DashboardTable.vue.orig [hunk 5]
```diff
@@ -0,0 +1,195 @@
+<template>
```
Comment (asvae) [inline src/components/dashboard/DashboardTable.vue.orig]: orig 3

File: src/components/maps/line-maps/LineMap.vue.orig [hunk 6]
```diff
@@ -0,0 +1,91 @@
+<template>
```
Comment (asvae) [inline src/components/maps/line-maps/LineMap.vue.orig]: orig

File: src/components/ui/icons/IconSet.vue.orig [hunk 7]
```diff
@@ -0,0 +1,192 @@
+<template>
```
Comment (asvae) [inline src/components/ui/icons/IconSet.vue.orig]: orig

File: src/vuestic-theme/vuestic-components/vuestic-layout/VuesticLayout.vue.orig [hunk 8]
```diff
@@ -0,0 +1,87 @@
+<template>
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-layout/VuesticLayout.vue.orig]: orig

File: src/app/main.js (hunk 1 above)
Comment (Kreezag) [reply src/app/main.js]: fixed

**Pull request merged**